enterprise/server/util/oci/oci.go

```diff
@@ -110,3 +121,107 @@ func (c Credentials) String() string {
 func (c Credentials) Equals(o Credentials) bool {
 	return c.Username == o.Username && c.Password == o.Password
 }
+
+func Resolve(ctx context.Context, imageName string, platform *rgpb.Platform, credentials Credentials) (v1.Image, error) {
```

Comment:
to simplify the API of this package, wdyt about having `Resolve` call `authorize()` as an implementation detail, rather than exposing `Authorized` as a separate func?

Reply:
Removed this function per above comment.